rt/cmake/cmake.tdesc

```diff
@@ -0,0 +1,102 @@
+-- -*- lua -*-
+
+-- Skip the CMake test if we can't find a new enough version fo CMake
+local skip_test = true
+local cmake_path = "/bin/false"
+local cmake_ret = os.execute("cmake --version > cmake_version.txt 2>&1")
```

Comment:
The `capabilities` command isn't sufficient.  The requirement for 3.6.0 is the usage of the FILTER sub command for list, which wasn't present until 3.6.  Whether we parse the version in lua or use a cmake script with return code, I suppose either way, something is dumping to a file and parsing for the version check.  At least using the suggestged cmake -P method, the version will be parsed in the same way by cmake as in the init script.  I'll adjust accordingly.